fendermint/vm/interpreter/src/fvm/state/fevm.rs

```diff
@@ -64,6 +64,10 @@ pub struct ContractCallerReturn<T> {
 
 impl<T: Detokenize> ContractCallerReturn<T> {
     pub fn into_decoded(self) -> anyhow::Result<T> {
+        self.decoded_ret()
+    }
+
+    pub fn decoded_ret(&self) -> anyhow::Result<T> {
```

Comment:
```suggestion
        self.decoded()
    }

    pub fn decoded(&self) -> anyhow::Result<T> {
```

If we already call it `into_decoded` and the only difference is whether it takes `self` or `&self` then I don't see a reason to introduce a different name what this can be turned into.